index.js

```diff
@@ -99,6 +100,11 @@ module.exports = function (text, opts) {
 
 	var NL = '\n';
 	var PAD = ' ';
+
+	if (opts.align && opts.align !== 'left') {
+		text = ansiAlign(text, {align: opts.align});
+	}
```

Comment:
Would be nice if `ansiAlign` could just handle the `left` option too. That way we could just do a straight passthrough:

```js
text = ansiAlign(text, {align: opts.align});
```

No `if` statement.